CONTRIBUTING.md

```diff
@@ -61,11 +61,12 @@ install `pytest`:
 pip install pytest
 ```
 
-And then run `pytest` either for server or tools test, or even combined:
+And then run `pytest` either for server, tools, or integrations tests, or even combined:
 
 ```sh
 pytest tabpy-server/server_tests/ --cov=tabpy-server/tabpy_server
 pytest tabpy-tools/tools_tests/ --cov=tabpy-tools/tabpy_tools --cov-append
+pytest tabpy-tools/tests/integration_tests/ --cov=models --cov-append
```

Comment:
Might want to note that the integration tests have side effects.

Reply:
@sbabayan I think instead of documenting it and make people to revert `state.ini` after every local test run the tests should point TabPy to a temporary `state.ini` in the same way we do for config and passwords files in unit tests.

@jnegara Great catch, we all overlooked this.

Reply:
@0golovatyi Even if we implement what you propose, the test would still install Python packages that are not necessary for running vanilla TabPy to the machine on which it is running. This should be documented.

Reply:
Agree. Packages we add to Python env is unrelated issue IMO and needs to be documented per model.